js_modules/dagster-ui/packages/ui-core/src/ticks/EvaluateTickButtonSchedule.tsx

```diff
@@ -0,0 +1,41 @@
+import {Box, Button} from '@dagster-io/ui-components';
+import {useState} from 'react';
+
+import {EvaluateScheduleDialog} from './EvaluateScheduleDialog';
+import {RepoAddress} from '../workspace/types';
+
+interface EvaluateTickButtonScheduleProps {
+  name: string;
+  repoAddress: RepoAddress;
+  jobName: string;
+}
+
+export const EvaluateTickButtonSchedule = ({
+  name,
+  repoAddress,
+  jobName,
+}: EvaluateTickButtonScheduleProps) => {
+  const [showTestTickDialog, setShowTestTickDialog] = useState(false);
+
+  return (
+    <Box flex={{direction: 'row', alignItems: 'center', gap: 8}}>
+      <Button
+        onClick={() => {
+          setShowTestTickDialog(true);
+        }}
+      >
+        Evaluate tick
+      </Button>
+      <EvaluateScheduleDialog
+        key={showTestTickDialog ? '1' : '0'} // change key to reset dialog state
```

Comment:
in retrospect i think this approach causes the closing animation not to play, not a big deal and out of scope for this PR but the solution is to break up the dialog into a wrapper Dialog / and inner DialogContent component